electric-operator.el

```diff
@@ -325,9 +325,8 @@ if not inside any parens."
 
 (i.e. takes one argument). This is a bit of a fudge based on C-like syntax."
   (or
-   (looking-back-locally "^\\s-*")
```

Comment:
Can you revert this removal please? I this function is used in a lot of different places so I'd rather not remove functionality from it because it might break something. 

We should be able to get the same effect by moving the `((looking-back-locally "^\\s-*") "+")` lines before `((probably-unary-operator?) " +")` in the various `cond` statements.

Reply:
I checked each use of that function to make sure it has no unintended side-effects but don't mind changing it if you prefer that

Reply:
Oh ok, should be fine to leave it then.